diff --git a/dist/api/package.js b/dist/api/package.js
index 8fb0bea8296b4426389441e2ade33d976262f865..b46e667703a681ec4b65405790fa751303ffd092 100644
--- a/dist/api/package.js
+++ b/dist/api/package.js
@@ -48,33 +48,26 @@ function hidePromiseFromPromisify(fn) {
  */
 function sequentialHooks(hooks) {
     return [
-        hidePromiseFromPromisify(async (buildPath, electronVersion, platform, arch, done) => {
+        async ({ buildPath, electronVersion, platform, arch }) => {
             for (const hook of hooks) {
                 try {
                     await (0, node_util_1.promisify)(hook)(buildPath, electronVersion, platform, arch);
                 }
                 catch (err) {
                     d('hook failed:', hook.toString(), err);
-                    return done(err);
+                    throw err;
                 }
             }
-            done();
-        }),
+        },
     ];
 }
 function sequentialFinalizePackageTargetsHooks(hooks) {
     return [
-        hidePromiseFromPromisify(async (targets, done) => {
+        async (targets) => {
             for (const hook of hooks) {
-                try {
-                    await (0, node_util_1.promisify)(hook)(targets);
-                }
-                catch (err) {
-                    return done(err);
-                }
+                await (0, node_util_1.promisify)(hook)(targets);
             }
-            done();
-        }),
+        },
     ];
 }
 const listrPackage = (childTrace, { dir: providedDir = process.cwd(), interactive = false, arch = (0, get_1.getHostArch)(), platform = process.platform, outDir, }) => {
