diff --git a/index.js b/index.js
index 94b61425f38d7f075ad41b6186905ab6d52dd994..beb560127cbb4986c28ce609f9b06ca05ad07a30 100644
--- a/index.js
+++ b/index.js
@@ -46,7 +46,7 @@ function zip (inPath, outPath, cb) {
   // Windows zip command does not overwrite existing files. So do it manually first.
   function doZip () {
     if (process.platform === 'win32') {
-      fs.rmdir(outPath, { recursive: true, maxRetries: 3 }, doZip2)
+      fs.rm(outPath, { recursive: true, force: true, maxRetries: 3 }, doZip2)
     } else {
       doZip2()
     }
@@ -72,7 +72,7 @@ function zipSync (inPath, outPath) {
       fs.writeFileSync(path.join(tmpPath, path.basename(inPath)), inFile)
       inPath = tmpPath
     }
-    fs.rmdirSync(outPath, { recursive: true, maxRetries: 3 })
+    fs.rmSync(outPath, { recursive: true, force: true, maxRetries: 3 })
   }
   const opts = {
     cwd: path.dirname(inPath),
